<ui version="4.0" >
 <class>CLogGuiComp</class>
 <widget class="QWidget" name="CLogGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="LogView" >
     <property name="editTriggers" >
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Message</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Source</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="CategoryCombo" >
       <item>
        <property name="text" >
         <string>Information</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Warning</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Critical</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>System</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Clear</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExportButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Export...</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections/>
</ui>
